bin/build.mjs

```diff
@@ -123,7 +123,7 @@ async function build() {
 
 		// Step 7: Build packages
 		console.log( '\n📦 Building packages (production mode)...' );
-		await exec( 'node', [ './bin/packages/build.mjs' ], {
+		await exec( 'wp-build', [], {
```

Comment:
I'm surprised this doesn't need to be executed through `npm exec` / `npx` ? How is `wp-build` on the `PATH` ?